pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -324,3 +345,19 @@ func (m *podContainerManagerNoop) GetAllPodsFromCgroups() (map[types.UID]CgroupN
 func (m *podContainerManagerNoop) IsPodCgroup(cgroupfs string) (bool, types.UID) {
 	return false, types.UID("")
 }
+
+func (m *podContainerManagerNoop) GetPodCgroupMemoryLimit(_ *v1.Pod) (uint64, error) {
```

Comment:
I probably have asked this before in another PR. why these functions returning constants?

Reply:
these are no-op implementations. all other no-op interfaces are doing the same.